src/Client/ClientBase.cpp

```diff
@@ -2857,6 +2857,12 @@ bool ClientBase::processQueryText(const String & text)
             return true;
         }
 
+        // Check if AI provider usage needs acknowledgment from user
```

Comment:
We can also improve `AI SQL generator is not initialized` message above.
Now we can at least put info about `OPENAI_API_KEY/ANTHROPIC_API_KEY` and refer to the doc for more sophisticated setup